api/src/graphql/connectors/index.js

```diff
@@ -287,3 +297,28 @@ export function searchRealities(driver, label, term) {
   `;
   return runQueryAndGetRecords(driver.session(), query, { term });
 }
+
+export function getPeopleTwoStepsFromReality(driver, { nodeId }) {
+  const query = `
+    MATCH (n {nodeId:'${nodeId}'})<-[*0..2]-(p:Person) 
+    WITH collect(distinct p) as pe
+    UNWIND pe as people
+    RETURN people
+  `;
+  return runQueryAndGetRecords(driver.session(), query, { nodeId });
+}
+
+export function getRealityData(driver, { nodeId }) {
+  const query = `
+  MATCH (n {nodeId:'${nodeId}'})
```

Comment:
Indent to be consistent with other queries

Reply:
Done